.gitlab-ci.yml

```diff
@@ -71,15 +78,36 @@ check-line-width:
   allow_failure:                   false
 
 
-test-linux-stable:                 &test
+cargo-audit:
+  stage:                           test
+  <<:                              *compiler_info
+  only:                            *not-prs
+  script:
+    - cargo audit
+  allow_failure:                   true
+  tags:
+    - linux-docker
+
+
+test-build-subkey:
+  stage:                           test
+  <<:                              *compiler_info
+  only:                            *not-prs
+  script:
+    - cd ./subkey
+    - time cargo build
```

Comment:
as substrate is build with `--release`, why not here as well?

Reply:
I was going to add it later, in #2478